### PR TITLE
Making sure backends don't expire for v2 annotations publish pipeline

### PR DESCRIPTION
**Pull request by catalin-ilea**

### CODE REVIEW

File: concept-suggestions-kafka-bridge-sidekick@.service [hunk 1]
```diff
@@ -4,15 +4,18 @@ BindsTo=concept-suggestions-kafka-bridge@%i.service
...
 ExecStart=/bin/sh -c "\
-  etcdctl set   /ft/services/concept-suggestions-kafka-bridge/healthcheck     true; \
-  etcdctl mkdir /ft/services/concept-suggestions-kafka-bridge/servers; \
-  etcdctl set /ft/healthcheck/concept-suggestions-kafka-bridge-%i/path /__health; \
-  while true; do \
-    CONCEPT_KAFKA_BRIDGE_PORT=`[ $CONCEPT_KAFKA_BRIDGE_PORT ] && echo $CONCEPT_KAFKA_BRIDGE_PORT || echo $(/usr/bin/docker port concept-suggestions-kafka-bridge-%i 8080 | cut -d':' -f2)`; \
-    etcdctl set /ft/services/concept-suggestions-kafka-bridge/servers/%i "http://$HOSTNAME:$CONCEPT_KAFKA_BRIDGE_PORT" --ttl 600; \
-    sleep 120; \
-  done"
+  export SERVICE=$(echo %p | sed "s/-sidekick//g"); \
```
Comment (Zapadlo) [inline concept-suggestions-kafka-bridge-sidekick@.service]: Unescaped double quotes

Comment (catalin-ilea) [inline concept-suggestions-kafka-bridge-sidekick@.service]: Thanks...I why this is working who. https://github.com/Financial-Times/up-service-files/blob/master/public-annotations-api-sidekick%40.service

Comment (catalin-ilea) [inline concept-suggestions-kafka-bridge-sidekick@.service]: ^^ has been in prod for some time now, haven't noticed any issues

**Comment by BerniVarga** [comment]
+1

**Comment by kovacshuni** [comment]
+1

**Comment by tosan88** [comment]
:+1:

**Comment by catalin-ilea** [comment]
@tamas-molnar added the changes you suggested

**Comment by tamas-molnar** [comment]
+1

**Comment by tosan88** [comment]
:+1:

**Comment by Zapadlo** [comment]
+1

**Pull request merged**